Source/WebCore/platform/graphics/FontRanges.h

```diff
@@ -61,7 +61,7 @@ class FontRanges {
 
         UChar32 from() const { return m_from; }
         UChar32 to() const { return m_to; }
-        const Font* font(ExternalResourceDownloadPolicy) const;
+        WEBCORE_EXPORT const Font* font(ExternalResourceDownloadPolicy) const;
```

Comment:
Why this change? Could you discard this?

Reply:
This change is because the above change in `FontCascadeFonts.h` is inlined, which means other frameworks will now try to call `FontRanges::font()`.